transfer_nlp/plugins/config.py

```diff
@@ -204,12 +204,18 @@ def recursive_replace(my_item: Union[Dict, List, str]):
 class ExperimentConfig:
 
     @staticmethod
-    def load_experiment_json(experiment: Union[str, Path, Dict]) -> Dict:
+    def load_experiment_dict(experiment: Union[str, Path, Dict]) -> Dict:
```

Comment:
That function doesn't expect a dict, maybe it should be named load_experiment ?

Reply:
Yeah, I've written dict like "load the experiment in a dictionary". We can name it `load_experiment` but is it clear then that it is not configured yet?

Reply:
mmh `load_experiment_to_dict` or `load_experiment_config` ?

Reply:
`load_experiment_config` seems good!